resources/views/components/input-field/checkbox.blade.php

```diff
@@ -0,0 +1 @@
+@include('rapidez::components.input-field.index', ['type' => 'checkbox'])
```

Comment:
Is the name "input" not enough? why add "field" behind it.

Reply:
Two reasons:
 - We wanted to have a separate input/label component as well as this one which combines both, giving people the freedom to build things up on their own
 - If you call a composite component like this just `input` then it becomes confusing, as you'd expect the base element of this component to be an input.